Bio/Alphabet/__init__.py

```diff
@@ -6,499 +6,17 @@
-            pass
-        elif stop is None:
-            stop = alpha.stop_symbol
-        elif stop == alpha.stop_symbol:
-            pass
-        else:
-            raise ValueError("More than one stop symbol present")
-        # New letters...
-        if hasattr(alpha, "new_letters"):
-            for letter in alpha.new_letters:
-                if letter not in new_letters and letter != gap and letter != stop:
-                    new_letters += letter
-
-    alpha = base
-    if new_letters:
-        alpha = AlphabetEncoder(alpha, new_letters)
-    if gap:
-        alpha = Gapped(alpha, gap_char=gap)
-    if stop:
-        alpha = HasStopCodon(alpha, stop_symbol=stop)
-    return alpha
-
-
-def _check_type_compatible(alphabets):
-    """Return True except for DNA+RNA or Nucleotide+Protein (PRIVATE).
-
-        >>> _check_type_compatible([generic_dna, generic_nucleotide])
-        True
-        >>> _check_type_compatible([generic_dna, generic_rna])
-        False
-        >>> _check_type_compatible([generic_dna, generic_protein])
-        False
-        >>> _check_type_compatible([single_letter_alphabet, generic_protein])
-        True
-
-    This relies on the Alphabet subclassing hierarchy.  It does not
-    check things like gap characters or stop symbols.
-    """
-    dna, rna, nucl, protein = False, False, False, False
-    for alpha in alphabets:
-        a = _get_base_alphabet(alpha)
-        if isinstance(a, DNAAlphabet):
-            dna = True
-            nucl = True
-            if rna or protein:
-                return False
-        elif isinstance(a, RNAAlphabet):
-            rna = True
-            nucl = True
-            if dna or protein:
-                return False
-        elif isinstance(a, NucleotideAlphabet):
-            nucl = True
-            if protein:
-                return False
-        elif isinstance(a, ProteinAlphabet):
-            protein = True
-            if nucl:
-                return False
-    return True
-
-
-def _verify_alphabet(sequence):
-    """Check all letters in sequence are in the alphabet (PRIVATE).
-
-    This replaces Bio.utils.verify_alphabet() since we are deprecating
-    that. Potentially this could be added to the Alphabet object, and
-    I would like it to be an option when creating a Seq object... but
-    that might slow things down.
-    """
-    letters = sequence.alphabet.letters
-    if not letters:
-        raise ValueError("Alphabet does not define letters.")
-    for letter in sequence:
-        if letter not in letters:
-            return False
-    return True
+raise RuntimeError("Bio.Alphabet has been removed from Biopython. In many cases, the alphabet can simply be ignored and removed from scripts. In a few cases, you may need to specify the ``molecule_type`` as an annotation on a SeqRecord for your script to work correctly. Please see https://github.com/biopython/biopython/issues/3156 for more information.")
```

Comment:
Wouldn't it be clearer to raise ``ImportError`` here?

That would make sense with usage as in the examples on #3156 wouldn't it?

Reply:
Also rather than pointing at the GitHub issue, I was assuming we'd put those examples in the DEPRECATED file and/or Tutorial somewhere.

Reply:
I am OK to put those examples somewhere, but preferably not the tutorial because explaining how things were done in previous versions gets confusing quickly especially for new users. The DEPRECATED file then again will not be seen if a binary installer is used, e.g. on Windows. Maybe we can put those examples on the Biopython website somewhere?

Reply:
The DEPRECATED file does get overlooked. Our website isn't a bad idea (although long term I'd prefer all the documentation be in one place - combining our current LaTeX tutorial content and API documentation via Sphinx)

I was also thinking about an OBF blog post about this - probably preceding the actual release.